setup.py

```diff
@@ -30,6 +30,10 @@
         'zip',
         '2a616cbb5b15aec8238f22c0d62dede1b6d155798adc45ff4d0206395a8a5833',
     ),
+    'cygwin': (
+        'zip',
+        '2a616cbb5b15aec8238f22c0d62dede1b6d155798adc45ff4d0206395a8a5833',
+    ),
 }
```

Comment:
perhaps to make this easier to update, do:
```python
POSTFIX_SHA256['cygwin'] = POSTFIX_SHA256['win32']
```

Reply:
good idea i completely missed that last night